DocumentDataAPI/DocumentDataAPI/Controllers/DocumentController.cs

```diff
@@ -0,0 +1,208 @@
+﻿using DocumentDataAPI.Data.Repositories;
+using DocumentDataAPI.Models;
+
+using Microsoft.AspNetCore.Mvc;
+
+using System.Net.Mime;
+
+namespace DocumentDataAPI.Controllers
+{
+    [ApiController]
+    [Route("document")]
```

Comment:
For at være consistent med endpoints i andre controllers
```suggestion
    [Route(RoutePrefixHelper.Prefix + "/documents")]
```